packages/nimiq-css/src/css/typography.css

```diff
@@ -573,6 +574,14 @@
       > *:not(summary) {
         padding: 0 1rem 14px 1rem;
       }
+
+      p {
+        margin: 12px;
```

Comment:
Since a few months ago, I prefer using `lh` for margins in typography blocks.

Read more here: https://webkit.org/blog/16831/line-height-units/

We can use 0.75lh alternative. What do you think?